pypif/util/pif_encoder.py

```diff
@@ -18,5 +20,11 @@ def default(self, obj):
             return []
         elif isinstance(obj, list):
             return [i.as_dictionary() for i in obj]
-        else:
+        elif hasattr(obj, 'as_dictionary'):
             return obj.as_dictionary()
+        elif isinstance(obj, (np.int_, np.intc, np.intp, np.int8, np.int16, np.int32,
+                              np.int64, np.uint8, np.uint16, np.uint32, np.uint64)):
+            return int(obj)
+        elif isinstance(obj, (np.float_, np.float16, np.float32, np.float64)):
+            return float(obj)
```

Comment:
Why not just cast to float? Is there a reason to maintain the int/float distinction during serialization?